include/RED4ext/Process.hpp

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <RED4ext/CString.hpp>
+#include <RED4ext/Common.hpp>
+#include <cstdint>
+
+namespace RED4ext
+{
+struct Process
+{
+    static constexpr uint32_t DefaultTimeout = 60000;
+
+    struct FixedWString
+    {
+        uint32_t length;
+        uint32_t maxLength;
+        const wchar_t* str;
+    };
```

Comment:
Shouldn't it be `WString` or `WideString` or `CWideString`? Capacity field suggests that this is a mutable string.

Reply:
Yeah, you're right - it looks like this is used elsewhere too. I'll do some digging and see what I can find.